docs/ReleaseInstructions.md

```diff
@@ -51,6 +51,7 @@
 * Cut a release branch with format `0.x.y` (no v) - this branch is required to publish docs to versioned path
   * Clone from master branch
   * Add 1 commit to the release branch to remove pre-release warnings and update install instructions to remove `--pre`: [Old diff](https://github.com/autogluon/autogluon/commit/1d66194d4685b06e884bbf15dcb97580cbfb9261)
+  * Add 1 commit that converts notebook links from `master` to `stable` by running this command from the root project directory: `LC_ALL=C find docs/tutorials/ -type f -exec sed -i '' 's#blob/master/docs#blob/stable/docs#' {} +`
```

Comment:
`LC_ALL=C` is used because of [this](https://stackoverflow.com/questions/19242275/re-error-illegal-byte-sequence-on-mac-os-x) and explain [here](https://unix.stackexchange.com/questions/87745/what-does-lc-all-c-do).